### PR TITLE
Don't require checkers to be in a package (Fixes #854)

### PR DESCRIPTION
**Pull request by smillst**
(Fixes #854)

### CODE REVIEW

File: framework/jtreg/issue845/checker/NotInPackageChecker.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.common.basetype.BaseTypeChecker;
+
+/**
+ * Checker that is in the default package.  This tests for Issue 845
+ * https://github.com/typetools/checker-framework/issues/845
```
Comment (wmdietl) [inline framework/jtreg/issue845/checker/NotInPackageChecker.java]: Small comment: we usually have this comment about which issue it is in the first lines of the file.
For JTReg tests, I would suggest having the link with the test driver, Test.java in this case.

**Comment by wmdietl** [comment]
Looks good!

File: framework/jtreg/issue845/checker/NotInPackageChecker.java (hunk 1 above)
Comment (smillst) [inline framework/jtreg/issue845/checker/NotInPackageChecker.java]: Test.java has the link in the `@summary` tag.  Would you prefer it else where?

**Comment by smillst** [comment]
@wmdietl Thanks for the quick review.

**Pull request merged**